src/styles/themes/ThemeColors.ts

```diff
@@ -0,0 +1,89 @@
+type Color = string;
+
+type ThemeColors = {
+    // Figma keys
+    appBG: Color;
+    splashBG: Color;
+    highlightBG: Color;
+    border: Color;
+    borderLighter: Color;
+    borderFocus: Color;
+    icon: Color;
+    iconMenu: Color;
+    iconHovered: Color;
+    iconSuccessFill: Color;
+    iconReversed: Color;
+    iconColorfulBackground: Color;
+    textSupporting: Color;
+    text: Color;
+    textColorfulBackground: Color;
+    link: Color;
+    linkHover: Color;
+    buttonDefaultBG: Color;
+    buttonHoveredBG: Color;
+    buttonPressedBG: Color;
+    danger: Color;
+    dangerHover: Color;
+    dangerPressed: Color;
+    warning: Color;
+    success: Color;
+    successHover: Color;
+    successPressed: Color;
+    transparent: Color;
+    signInPage: Color;
+
+    // Additional keys
+    overlay: Color;
+    inverse: Color;
+    shadow: Color;
+    componentBG: Color;
+    hoverComponentBG: Color;
+    activeComponentBG: Color;
+    signInSidebar: Color;
+    sidebar: Color;
+    sidebarHover: Color;
+    heading: Color;
+    textLight: Color;
+    textDark: Color;
+    textReversed: Color;
+    textBackground: Color;
+    textMutedReversed: Color;
+    textError: Color;
+    offline: Color;
+    modalBackdrop: Color;
+    modalBackground: Color;
+    cardBG: Color;
+    cardBorder: Color;
+    spinner: Color;
+    unreadIndicator: Color;
+    placeholderText: Color;
+    heroCard: Color;
+    uploadPreviewActivityIndicator: Color;
+    dropUIBG: Color;
+    receiptDropUIBG?: Color;
```

Comment:
Why some properties are optional? Shouldn't both themes have the same properties defined?

Reply:
Yes! That's why i asked about that in [this comment](https://github.com/Expensify/App/pull/24178#issuecomment-1725070941) :)

Reply:
It's just a temporary solution until we have consistent theme values across light and dark themes

Reply:
I think we should fix the inconsistency of themes, thanks for the link to that comment 👍

Reply:
Yes, let's fix inconsistency of themes. We should add a comment note that we should prevent adding any optional properties to the theme object